backend/src/apiserver/server/api_converter.go

```diff
@@ -148,14 +152,49 @@ func toApiParameters(paramsString string) ([]*api.Parameter, error) {
 	return apiParams, nil
 }
 
+func toApiRuntimeConfig(modelRuntime model.RuntimeConfig) (*api.PipelineSpec_RuntimeConfig, error) {
+	// Convert RuntimeConfig.Parameters
```

Comment:
nit: the comment doesn't carry much useful information, can be removed.

Reply:
removed